src/tests/ftest/scrubber/frequency.yaml

```
@@ -0,0 +1,67 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+timeout: 1800
+setup:
+  start_servers_once: false
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31516
+      log_file: daos_server1.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+  transport_config:
+    allow_insecure: true
+agent_config:
+  transport_config:
+    allow_insecure: true
+dmg:
+  transport_config:
+    allow_insecure: true
+pool:
+  scm_size: 6000000000
+  nvme_size: 54000000000
+  svcn: 4
+  control_method: dmg
+  rebuild_timeout: 120
+  pool_query_timeout: 30
+  properties: "scrub:timed"
+container:
+  type: POSIX
+  control_method: daos
+  oclass: RP_2G1
+ior:
+  ior_timeout: 60
+  client_processes:
+    np_16:
+      np: 2
+  test_file: daos:testFile
+  repetitions: 2
+  dfs_destroy: false
+  iorflags:
+    ior_flags:
+      - "-v -W -w -r -R -k"
+    ior_api:
+      - DFS
+    transfer_block_size:
+      - [1M, 32M]
+    obj_class:
+      - RP_2G1
```

We should simplify this
```suggestion
  flags: "-v -W -w -r -R -k"
  api: DFS
  transfer_size: 1M
  block_size: 32M
  dfs_oclass: RP_2G1
```